apps/site/components/Containers/NavBar/index.tsx

```diff
@@ -53,17 +53,21 @@ const NavBar: FC<NavbarProps> = ({
         <Link className={style.nodeIconWrapper} href="/" aria-label="Home">
           <WithNodejsLogo />
         </Link>
-
         <Label.Root
-          onClick={() => setIsMenuOpen(prev => !prev)}
           className={style.sidebarItemTogglerLabel}
           htmlFor="sidebarItemToggler"
         >
           {navInteractionIcons[isMenuOpen ? 'close' : 'show']}
         </Label.Root>
       </div>
 
-      <input className="peer hidden" id="sidebarItemToggler" type="checkbox" />
+      <input
+        className={`peer ${style.sidebarItemToggler}`}
```

Comment:
We use `classNames` for multiple class names

Reply:
OK, but FYI later in the same file we do not use `classNames` https://github.com/nodejs/nodejs.org/blob/42d2a3ce45e729002dd6c74cdfeba5267dbf7978/apps/site/components/Containers/NavBar/index.tsx#L68

Reply:
Could you fix that? 🙏